scripts/fsutil.sh

```diff
@@ -0,0 +1,320 @@
+#!/bin/bash
+## This script is to create the tmpfs for different mount point during 
+## the ONIE installer image generation to speed up the processing.
+##
+## USAGE:
+##   ./tmpfs <storage> <operation>
```

Comment:
The description looks like out of date, right? Change ./tmpfs to fsutil.sh? And the script does not only handle the tmpfs for different mount point, right?